README.md

```diff
@@ -120,20 +120,33 @@ Alternatively, use online services (like Google Colab):
 1. Install the dependencies:
 ```bash
 # Debian-based:
-sudo apt install wget git python3 python3-venv libgl1 libglib2.0-0
+sudo apt install wget git python3 python3-venv libgl1 libglib2.0-0 make dpkg
 # Red Hat-based:
-sudo dnf install wget git python3 gperftools-libs libglvnd-glx 
+sudo dnf install wget git python3 gperftools-libs libglvnd-glx make dpkg
 # openSUSE-based:
-sudo zypper install wget git python3 libtcmalloc4 libglvnd
+sudo zypper install wget git python3 libtcmalloc4 libglvnd make dpkg
 # Arch-based:
-sudo pacman -S wget git python3
+sudo pacman -S wget make dpkg git python3
 ```
-2. Navigate to the directory you would like the webui to be installed and execute the following command:
+2. Download: https://www.python.org/ftp/python/3.10.6/Python-3.10.6.tar.xz
+
+3. extract using ark or any file archive tool
+
+```
+cd $HOME/python-3.10.6
+
+./configure
+
+make
+
+sudo make install

```

Comment:
installing python 3.10 like this might cause all kinds of side effects since it ends up in /usr/local and might be chosen as the default python.

Reply:
Mate, thats the point is for SD to install as main python cause it works best for SD

Reply:
However SD is not the only python application on that distribution. If 3.10 is not the default python but you install it in this way, this can break any other python application that might require a different version. E.g. it might have the side effect that you cannot login anymore or some parts of your window manager dont work anymore.

If python 3.10 is not the default, it might be better to install it either using conda or through a package manager.

Reply:
Mate, the way I set it to install it wont be installed as default unless you install it before the dep commands in which case its a you mistake

Reply:
Not sure I can follow. If I do as layed out here in a standard docker environment, then this will for sure overshadow the default python. Moreover, the list of dependencies required to build python3 from source is far greater than only `make` and `dpkg`.